distribution/docs/src/main/resources/_contents/_configuring/configuring-from-configuration-files-contents.adoc

```diff
@@ -0,0 +1,746 @@
+|jks
+|Yes
+
+6+^h| Global URL Properties
+
+|Protocol
+|org.codice.ddf.system.protocol
+|String
+|Default protocol that should be used to connect to this machine.
+|https://
+|Yes
+
+|Hostname
+|org.codice.ddf.system.hostname
+|String
+|The hostname or IP address used to advertise the system. Do not enter `localhost`. Possibilities include the address of a single node or that of a load balancer in a multi-node deployment.
+
+NOTE: Does not change the address the system runs on.
+|localhost
+|Yes
+
+|HTTPS Port
+|org.codice.ddf.system.httpsPort
+|String
+|The https port used by the system.
+
+NOTE: This *DOES* change the port the system runs on.
+|8993
+|Yes
+
+|HTTP Port
+|org.codice.ddf.system.httpPort
+|String
+|The http port used by the system.
+
+NOTE: This *DOES* change the port the system runs on.
+|8181
+|Yes
+
+|Default Port
+|org.codice.ddf.system.port
+|String
+|The default port used to advertise the system. This should match either the http or https port.
+
+NOTE: Does not change the port the system runs on.
+|8993
+|Yes
+
+|Root Context
+|org.codice.ddf.system.rootContext
+|String
+|The the base or root context that services will be made available under.
+|/services
+|Yes
+
+6+^h|System Information Properties
+
+|Site Name
+|org.codice.ddf.system.id
+|String
+|The site name for ${branding}.
+|ddf.distribution
+|Yes
+
+|Site Contact
+|org.codice.ddf.system.siteContact
+|String
+|The email address of the site contact.
+|
+|No
+
+|Version
+|org.codice.ddf.system.version
+|String
+|The version of ${branding} that is running.
+
+This value should not be changed from the factory default.
+|${project.version}
+|Yes
+
+|Organization
+|org.codice.ddf.system.organization
+|String
+|The organization responsible for this installation of ${branding}.
+|Codice Foundation
+|Yes
+
+6+^h|Thread Pool Settings
+
+|Thread Pool Size
+|org.codice.ddf.system.threadPoolSize
+|Integer
+|Size of thread pool used for handling UI queries, federating requests, and downloading resources
+|128
+|Yes
+
+6+^h|HTTPS Specific Settings
+
+|Cipher Suites
+|https.cipherSuites
+|String
+|Cipher suites to use with secure sockets
+|TLS_DHE_RSA_WITH_AES_128_CBC_SHA,
+
+TLS_DHE_RSA_WITH_AES_128_CBC_SHA,
+
+TLS_DHE_DSS_WITH_AES_128_CBC_SHA,
+
+TLS_RSA_WITH_AES_128_CBC_SHA
+|No
+
+|Https Protocols
+|https.protocols
+|String
+|Protocols to allow for secure connections
+|TLSv1.1,TLSv1.2
+|No
+
+|Allow Basic Auth Over Http
+|org.codice.allowBasicAuthOverHttp
+|Boolean
+|Set to true to allow Basic Auth credentials to be sent over HTTP unsecurely. This should only be done in a test environment. These events will be audited.
+|false
+|Yes
+
+|Restrict the Security Token Service to allow connections only from DNs matching these patterns
+|ws-security.subject.cert.constraints
+|String
+|Set to a comma separated list of regex patterns to define which hosts are allowed to connect to the STS
+|.*
+|Yes
+
+6+^h|XML Settings
+
+|Parse XML documents into DOM object trees
```

Comment:
I do not think we should be documenting system properties that we did not add or we expect admins to ever change.  Some of these come from Karaf and are not expected to be changed.